client/data/users/use-users-query.js

```diff
@@ -38,10 +38,19 @@ const useUsersQuery = ( siteId, fetchOptions = {}, queryOptions = {} ) => {
 			 * such as Administrator and Editor, and has also been added as a "Viewer" .
 			 */
 			const users = uniqueBy( extractPages( data.pages ), compareUnique );
+			const found = users?.length ?? data.pages[ 0 ].found;
+			const totalUsers = data.pages?.[ 0 ]?.total_users || found;
 			return {
-				users: uniqueBy( extractPages( data.pages ), compareUnique ),
-				total: users?.length ?? data.pages[ 0 ].found,
```

Comment:
This fixes something I missed in https://github.com/Automattic/wp-calypso/pull/99448. There are duplicate calls to `uniqueBy( extractPages( data.pages ), compareUnique )`.

It also rearranges the destructuring so that the values we want to overwrite are actually overwritten.